Add content to articles.html

luy17 opened the pull request:
Work on the collapse function and start working on copying and pasting the different articles.

luy17 committed:
Add content to articles.html

Work on the collapse function and start working on copying and pasting the different articles.

callumhauber commented on articles.html:
```diff
@@ -3,18 +3,243 @@
+    <h1 class="head">The 10 Articles of the US Constitution</h1> 
+    <div id="articles">
+      <!--------------------------Article 1------------------------------>
+      <h3>Article I</h3>
+      <div>
+        <p><b>Section 1</b></p>
+        <p>
+          All legislative Powers herein granted shall be vested in a Congress of the United States, which shall consist of a Senate and House of Representatives.
+        </p>
```
I would use <section> tags to separate each article of the constitution, otherwise this looks good

callumhauber reviewed (commented):
I would use `<section>` tags to separate each article of the constitution, otherwise this looks good

luy17 committed:
Replaced div containers with section tags

luy17 committed:
CSS Formating

Worked on CSS for background and color scheme. Also worked on on a header for articles.html.

callumhauber commented:
Just so this is in the history: I'm approving this as-is, but this is a fork so it's kinda weird for us to be able to do stuff with the code in this PR since it's not a branch of the lab2 repo. In the future you just need to make a branch off lab2
